sympy/geometry/curve.py

```diff
@@ -232,12 +259,8 @@ def parameter(self):
         Returns
         =======
 
-        parameter : SymPy symbol
-
-        See Also
-        ========
-
-        functions
+        Symbol:
```

Comment:
```suggestion
        Symbol :
```
(not sure -- some have space and some don't)

Reply:
I think providing space to the docstrings will make it more readable.